.github/workflows/project.yml

```diff
@@ -147,3 +147,10 @@ jobs:
       - run: yarn type:test
       - run: yarn type:doc
 
+  verify-yaml:
```

Comment:
I think this (and the name of the script file too) could use a more descriptive name of what it's doing. Something like `verify-plugin-versions` or anything more descriptive.